src/app/components/Calendar/CalendarView.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import 'react-modern-calendar-datepicker/lib/DatePicker.css';
+import { Calendar } from 'react-modern-calendar-datepicker';
+import styled from 'styled-components';
+import './styles.css';
+
+export default function CalendarView(): JSX.Element {
+  const sessions = JSON.parse(localStorage.getItem('sessions') || '[]');
+  return (
+    <Container>
+      <Header>Your Meditation sessions</Header>
+      <Calendar
+        value={sessions}
+        colorPrimary="var(--green-background)"
+        calendarClassName="customCalendar"
+      />
+    </Container>
+  );
+}
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`;
+
```

Comment:
I guess you use the styles.css because <Calendar> is already a(n imported) component?
If you would like to use styled components, try something like the following:
 ```suggestion
const MyCalendar = styled(Calendar)`
   box-shadow: none;
`;
```
This is about [extending styles](https://styled-components.com/docs/basics#extending-styles)

Reply:
I agree with everything but the variable name :)

Reply:
thx to you both! @marwinburesch the problem is, that `<Calendar>` actually comes from the package I installed. Changing it through styled components actually will let the app crash. Unless you have any other suggestions https://kiarash-z.github.io/react-modern-calendar-datepicker/docs/customization